### DISCUSSION TITLE
SOLVED...Save Failed: CONT_0001 on Edit JSON on Data tab

### DISCUSSION THREAD

**Discussion by spoladian** [EsriPS/arcgis-assistant-feedback#16] [Q&A] [answered]
Solved.  Logged in with the owner vs with my admin account and was able to prevent the notice. 

Good morning,

I hoping that someone will be able to point me in the correct direction on figuring our what is going on or what I may be doing wrong.
I'm seeing an error / notice "Save failed: CONT_0001: Item does not exist or is inaccessible." on the bottom of the screen when I save JSON edits in the data tab of the new assistant.  This app was originally built using web app builder.

The error / notice continues to popup on save, though the changes are being saved and the app is working.

![image](https://user-images.githubusercontent.com/5775186/129041191-d25470dc-c3a5-45e6-b7b0-67ad3abe18b9.png)

Any help would be appreciated.

Thank you,

Steve Poladian
City of Cape Coral

**Comment by jpeterson** [comment]
Hi @spoladian! I was just about to ask for a little more info, but I see you resolved your issue by switching to another user – very glad you got it sorted!

You should be able to perform this action as an admin, so the error message (which is coming from ArcGIS Online) is still curious. If you have a few moments, would you mind following these steps while logged in as your admin user so we can see about getting this issue fixed?

1. Open your browser's developer tools and select the "Network" tab (F12 on Windows, or you can right-click the page and select "Inspect").

2. Perform your JSON edits and click save

3. Select the request called "update"

4. Screenshot and share the result from the "Preview" tab in the right panel.

![image](https://user-images.githubusercontent.com/3399064/129049914-51ef7000-7209-4370-97ed-224cf0439dee.png)

**Reply by bren96** [reply]
@jpeterson Running into the same problem. I am an admin for the AGOL Organization and am trying to edit the JSON of an item that I am not the owner of. I can edit the Description JSON, but cannot change any of the Data JSON.

I followed your steps and this is what my network preview showed:
```
error: {code: 400, messageCode: "CONT_0001", message: "Item does not exist or is inaccessible.", details: []}
code: 400
details: []
message: "Item does not exist or is inaccessible."
messageCode: "CONT_0001"
```

**Reply by jpeterson** [reply]
Thanks @bren96! It does look like there's a bug here. We'll look into it and see what comes up. 

In the meantime, it appears the best workaround is to login as the owner of the item. Depending on your scenario, you could transfer ownership of the item to your admin user (in Online), edit it (in Assistant), then transfer back to the original owner (again in Online).

**Reply by bren96** [reply]
@jpeterson Thanks for the update and workaround!

**Comment by spoladian** [comment]
Josh,

I'm performing the steps outlined and there is nothing being recorded in the devtools.  I also verified that I'm able to get devtool results on another page successfully.  I'd be more then happy to do a screen share with you to gather the information needed.  Let me know how we can proceed.

Thank you,

*** My email address has recently changed to ***@***.******@***.***> please ensure you are using this email address going forward. ***

Steve Poladian
Program Analyst - GIS Applications
* ***@***.******@***.***>
City of Cape Coral, ITS Dept.
1015 Cultural Park Blvd.
Cape Coral, Fl 33990

Did you know the best way to report issues to ITS is to use the Service Desk system?  This will ensure someone in ITS gets your request and allows us to track the progress.  Please enter all your issues and/or questions by clicking https://belaire.cape.capecoral.net<https://belaire.cape.capecoral.net/>

NOTE:  Florida has a very broad public records law, and under Florida law, most written communications to or from city staff regarding city business to include your e-mail address is considered public records and will be made available to the public and the media upon request. If you do not want your email message and or your e-mail address released in response to a public records request, do not send electronic mail to this entity. Instead, contact this office by phone or in writing.  Additionally, this communication is intended only for the addressee.  If you are not the intended recipient, do not copy, disclose, or distribute this message to anyone else.  If you have received this communication in error, please contact the sender of the message to inform him or her of the error and then delete this message.

From: Josh Peterson ***@***.***>
Sent: Wednesday, August 11, 2021 10:46 AM
To: EsriPS/arcgis-assistant-feedback ***@***.***>
Cc: Steve Poladian ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] - Re: [EsriPS/arcgis-assistant-feedback] SOLVED...Save Failed: CONT_0001 on Edit JSON on Data tab (#16)

 Caution - This email originated from outside of our organization. Please do not open any attachments or click on any links from unknown sources or unexpected email.
________________________________



Hi @spoladian<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fspoladian&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908336864%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=3UXsyoKzm9E%2F%2F5Y%2FPOawZKl9qxD8IhRh9TI37zPPxro%3D&reserved=0>! I was just about to ask for a little more info, but I see you resolved your issue by switching to another user - very glad you got it sorted!

You should be able to perform this action as an admin, so the error message (which is coming from ArcGIS Online) is still curious. If you have a few moments, would you mind following these steps while logged in as your admin user so we can see about getting this issue fixed?

  1.  Open your browser's developer tools and select the "Network" tab (F12 on Windows, or you can right-click the page and select "Inspect").
  2.  Perform your JSON edits and click save
  3.  Select the request called "update"
  4.  Screenshot and share the result from the "Preview" tab in the right panel.

<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F3399064%2F129049914-51ef7000-7209-4370-97ed-224cf0439dee.png&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908346822%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=41Ii5bGbURtGH5RvB5xwK%2Bv%2BC2YJxVKnijQAmLHETdQ%3D&reserved=0>

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FEsriPS%2Farcgis-assistant-feedback%2Fdiscussions%2F16%23discussioncomment-1159174&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908346822%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=GKWQ9MfIYx6bq1Ygv6rYeSh6eEfhbvGXVaZjVe6bk0w%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABMB6URG6RGGFWB52OZWFW3T4KEMHANCNFSM5B6SBS6Q&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908356757%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=CQZkxXdUpqy%2FhSEtrE0zNPrDa7LgXpByVZNFmqkDyBQ%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908356757%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=422E9w%2Fvzrco5oWHwcvsrzDCta7N885kbxKi56LhRCw%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908366733%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ZSRmC71oLWkKWjh1TQKZRiY79Zi0OIywL7pTSuhgiuI%3D&reserved=0>.

**Reply by jpeterson** [reply]
Thanks, Steve! The notes from you and @bren96 about this error only displaying when you're editing another user's item as an org admin helped us reproduce the issue on our end. We'll dig into it and work on a fix.

I'll update this thread once we have a resolution. Thanks again for letting us know!

**Comment by spoladian** [comment]
***@***.***

*** My email address has recently changed to ***@***.******@***.***> please ensure you are using this email address going forward. ***

Steve Poladian
Program Analyst - GIS Applications
* ***@***.******@***.***>
City of Cape Coral, ITS Dept.
1015 Cultural Park Blvd.
Cape Coral, Fl 33990

Did you know the best way to report issues to ITS is to use the Service Desk system?  This will ensure someone in ITS gets your request and allows us to track the progress.  Please enter all your issues and/or questions by clicking https://belaire.cape.capecoral.net<https://belaire.cape.capecoral.net/>

NOTE:  Florida has a very broad public records law, and under Florida law, most written communications to or from city staff regarding city business to include your e-mail address is considered public records and will be made available to the public and the media upon request. If you do not want your email message and or your e-mail address released in response to a public records request, do not send electronic mail to this entity. Instead, contact this office by phone or in writing.  Additionally, this communication is intended only for the addressee.  If you are not the intended recipient, do not copy, disclose, or distribute this message to anyone else.  If you have received this communication in error, please contact the sender of the message to inform him or her of the error and then delete this message.

From: Josh Peterson ***@***.***>
Sent: Wednesday, August 11, 2021 10:46 AM
To: EsriPS/arcgis-assistant-feedback ***@***.***>
Cc: Steve Poladian ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] - Re: [EsriPS/arcgis-assistant-feedback] SOLVED...Save Failed: CONT_0001 on Edit JSON on Data tab (#16)

 Caution - This email originated from outside of our organization. Please do not open any attachments or click on any links from unknown sources or unexpected email.
________________________________



Hi @spoladian<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fspoladian&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908336864%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=3UXsyoKzm9E%2F%2F5Y%2FPOawZKl9qxD8IhRh9TI37zPPxro%3D&reserved=0>! I was just about to ask for a little more info, but I see you resolved your issue by switching to another user - very glad you got it sorted!

You should be able to perform this action as an admin, so the error message (which is coming from ArcGIS Online) is still curious. If you have a few moments, would you mind following these steps while logged in as your admin user so we can see about getting this issue fixed?

  1.  Open your browser's developer tools and select the "Network" tab (F12 on Windows, or you can right-click the page and select "Inspect").
  2.  Perform your JSON edits and click save
  3.  Select the request called "update"
  4.  Screenshot and share the result from the "Preview" tab in the right panel.

<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F3399064%2F129049914-51ef7000-7209-4370-97ed-224cf0439dee.png&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908346822%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=41Ii5bGbURtGH5RvB5xwK%2Bv%2BC2YJxVKnijQAmLHETdQ%3D&reserved=0>

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FEsriPS%2Farcgis-assistant-feedback%2Fdiscussions%2F16%23discussioncomment-1159174&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908346822%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=GKWQ9MfIYx6bq1Ygv6rYeSh6eEfhbvGXVaZjVe6bk0w%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABMB6URG6RGGFWB52OZWFW3T4KEMHANCNFSM5B6SBS6Q&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908356757%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=CQZkxXdUpqy%2FhSEtrE0zNPrDa7LgXpByVZNFmqkDyBQ%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908356757%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=422E9w%2Fvzrco5oWHwcvsrzDCta7N885kbxKi56LhRCw%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cspoladia%40capecoral.gov%7C9a698301c8dd4ce3923c08d95cd6cce8%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637642899908366733%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ZSRmC71oLWkKWjh1TQKZRiY79Zi0OIywL7pTSuhgiuI%3D&reserved=0>.

**Comment by jpeterson** [comment, marked as answer]
The [16 August release](https://guide.assistant.esri-ps.com/docs/changelog#16-august-2021) should address the root cause of the issue @spoladian and @bren96 discuss above, so the workaround should be unnecessary. 

Please leave a comment here if anyone is still running into this!

**Comment by spoladian** [comment]
Josh,

Thank you for the prompt update.
What time do you anticipate the update to be published?  I just performed the same workflow and am still receiving the same notice.

Thank you,

*** My email address has recently changed to ***@***.******@***.***> please ensure you are using this email address going forward. ***

Steve Poladian
Program Analyst - GIS Applications
* ***@***.******@***.***>
City of Cape Coral, ITS Dept.
1015 Cultural Park Blvd.
Cape Coral, Fl 33990

Did you know the best way to report issues to ITS is to use the Service Desk system?  This will ensure someone in ITS gets your request and allows us to track the progress.  Please enter all your issues and/or questions by clicking https://belaire.cape.capecoral.net<https://belaire.cape.capecoral.net/>

NOTE:  Florida has a very broad public records law, and under Florida law, most written communications to or from city staff regarding city business to include your e-mail address is considered public records and will be made available to the public and the media upon request. If you do not want your email message and or your e-mail address released in response to a public records request, do not send electronic mail to this entity. Instead, contact this office by phone or in writing.  Additionally, this communication is intended only for the addressee.  If you are not the intended recipient, do not copy, disclose, or distribute this message to anyone else.  If you have received this communication in error, please contact the sender of the message to inform him or her of the error and then delete this message.

From: Josh Peterson ***@***.***>
Sent: Monday, August 16, 2021 10:29 AM
To: EsriPS/arcgis-assistant-feedback ***@***.***>
Cc: Steve Poladian ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] - Re: [EsriPS/arcgis-assistant-feedback] SOLVED...Save Failed: CONT_0001 on Edit JSON on Data tab (#16)

 Caution - This email originated from outside of our organization. Please do not open any attachments or click on any links from unknown sources or unexpected email.
________________________________



The 16 August release<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fguide.assistant.esri-ps.com%2Fdocs%2Fchangelog%2316-august-2021&data=04%7C01%7Cspoladia%40capecoral.gov%7Cb3c3115823e64c1af7f508d960c23adf%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647209599704377%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=AILsLh6ts5NIgSCFQnsJz3SUkLuGOq6YPZKQEerwBsU%3D&reserved=0> should address the root cause of the issue @spoladian<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fspoladian&data=04%7C01%7Cspoladia%40capecoral.gov%7Cb3c3115823e64c1af7f508d960c23adf%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647209599714332%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=TMhMky2q37rUoljn0G3Iyy7X%2BJcacz9KnJIMl1nsFHU%3D&reserved=0> and @bren96<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbren96&data=04%7C01%7Cspoladia%40capecoral.gov%7Cb3c3115823e64c1af7f508d960c23adf%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647209599714332%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=g4y6oi7MQEN4WUSr5GE5rCsOJskqjiI3gZPmySftOoo%3D&reserved=0> discuss above, so the workaround should be unnecessary.

Please leave a comment here if anyone is still running into this!

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FEsriPS%2Farcgis-assistant-feedback%2Fdiscussions%2F16%23discussioncomment-1190343&data=04%7C01%7Cspoladia%40capecoral.gov%7Cb3c3115823e64c1af7f508d960c23adf%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647209599724287%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=VKSTVL4fbRNf1G5i0FtF%2BFYckw66RF6Zi9Si6nqh7xw%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABMB6UXPDZOLM3JWBVJECM3T5EOD3ANCNFSM5B6SBS6Q&data=04%7C01%7Cspoladia%40capecoral.gov%7Cb3c3115823e64c1af7f508d960c23adf%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647209599724287%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=VfBks65PCJAheuB14aBZtZOzXaeod11diBUGOSeYfRc%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cspoladia%40capecoral.gov%7Cb3c3115823e64c1af7f508d960c23adf%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647209599734243%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=DXP81h17PRssMyEyhoz%2FPsdSEQcNT94yLsv8hhQO%2FwU%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cspoladia%40capecoral.gov%7Cb3c3115823e64c1af7f508d960c23adf%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647209599734243%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=x41UR38ZJfh4nphNm6JStnyqE%2BwWW1W2wVQKzCdERJs%3D&reserved=0>.

**Reply by jpeterson** [reply]
Hi Steve, the update is live now. You might need to clear your browser cache. You can verify you're using the latest release by the build number listed in the header (should be 20210816.1)

![image](https://user-images.githubusercontent.com/3399064/129592997-ba51accf-4ed8-48a0-b565-8a3184fe5d4c.png)

**Comment by spoladian** [comment]
Josh,

Thank you for the prompt change!  It works like a charm!

Thank you,

*** My email address has recently changed to ***@***.******@***.***> please ensure you are using this email address going forward. ***

Steve Poladian
Program Analyst - GIS Applications
* ***@***.******@***.***>
City of Cape Coral, ITS Dept.
1015 Cultural Park Blvd.
Cape Coral, Fl 33990

Did you know the best way to report issues to ITS is to use the Service Desk system?  This will ensure someone in ITS gets your request and allows us to track the progress.  Please enter all your issues and/or questions by clicking https://belaire.cape.capecoral.net<https://belaire.cape.capecoral.net/>

NOTE:  Florida has a very broad public records law, and under Florida law, most written communications to or from city staff regarding city business to include your e-mail address is considered public records and will be made available to the public and the media upon request. If you do not want your email message and or your e-mail address released in response to a public records request, do not send electronic mail to this entity. Instead, contact this office by phone or in writing.  Additionally, this communication is intended only for the addressee.  If you are not the intended recipient, do not copy, disclose, or distribute this message to anyone else.  If you have received this communication in error, please contact the sender of the message to inform him or her of the error and then delete this message.

From: Josh Peterson ***@***.***>
Sent: Monday, August 16, 2021 11:58 AM
To: EsriPS/arcgis-assistant-feedback ***@***.***>
Cc: Steve Poladian ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] - Re: [EsriPS/arcgis-assistant-feedback] SOLVED...Save Failed: CONT_0001 on Edit JSON on Data tab (#16)

 Caution - This email originated from outside of our organization. Please do not open any attachments or click on any links from unknown sources or unexpected email.
________________________________



Hi Steve, the update is live now. You might need to clear your browser cache. You can verify you're using the latest release by the build number listed in the header (should be 20210816.1)

[image]<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F3399064%2F129592997-ba51accf-4ed8-48a0-b565-8a3184fe5d4c.png&data=04%7C01%7Cspoladia%40capecoral.gov%7Ce5de368c75a24d6016ae08d960ce9353%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647262616460518%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=qKl2%2Fc56qNLhgveTAXADM7BkCgbVO%2FlfA%2BdhWXvmoHk%3D&reserved=0>

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FEsriPS%2Farcgis-assistant-feedback%2Fdiscussions%2F16%23discussioncomment-1190823&data=04%7C01%7Cspoladia%40capecoral.gov%7Ce5de368c75a24d6016ae08d960ce9353%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647262616460518%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=IYcICSBtW3cEdPFF6sgOD2yam2uEinJOF%2FyqgrcwJ%2Bg%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABMB6US73JEC3YSFD2DFZCDT5EYPHANCNFSM5B6SBS6Q&data=04%7C01%7Cspoladia%40capecoral.gov%7Ce5de368c75a24d6016ae08d960ce9353%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647262616470475%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=XL2M%2Bnd3jRanDCxI9o7EIeRFfg8sK%2FwmeXOSu6PZ8%2F8%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cspoladia%40capecoral.gov%7Ce5de368c75a24d6016ae08d960ce9353%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647262616470475%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=uumpJLskD58AiVByQdd8Alnnupm0pPhy9FsgzKuYCig%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cspoladia%40capecoral.gov%7Ce5de368c75a24d6016ae08d960ce9353%7Cbc36902e2e9e47d7bce3bf2d652121d1%7C0%7C0%7C637647262616480429%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=nBTuVY7wnZtVd30IhGC4wT7cIlFE%2BGQfI%2B8GpNOJrK0%3D&reserved=0>.

**Comment by ewagstaff** [comment]
Hi @jpeterson, this is not related to arcgis-assistant-feedback but I am getting this exact error when using ArcGIS online. We deleted a view within ArcGIS Online's GUI and when we tried to rebuild it, we got blocked by this error:

{"error":{"code":400,"message":"Item does not exist or is inaccessible.","messageCode":"CONT_0001","details":["Item does not exist or is inaccessible."]}}

It turns out you can see it too without even a proper token: https://services3.arcgis.com/gxsXHdk0MOVUX9bl/arcgis/rest/services/id11_DemoOrganization_((Alerts_Test))_view/FeatureServer/0?f=json

Any tips on what we can do to resolve this? Guessing it was the same problem as the one you resolved for arcgis-assistant.
